### ISSUE TITLE
PWA Assets generation on the fly

### ISSUE THREAD

**Issue by LoranRendel** [vite-pwa/vite-plugin-pwa#682]
``` plugins: [
    VitePWA({
      pwaAssets: {
        image: 'logo.png',
        includeHtmlHeadLinks: true,
        overrideManifestIcons: true
      },
    })
  ]
```

This config doesn't produce image files in build folder.
Please tell be what I doing wrong.

**Comment by userquin** [comment]
In dev server, the images are not generated, only when running the build command. If running the build command and png files not being generated I'll check what's the problem.

**Comment by LoranRendel** [comment]
Yes, I use `vite build` command.

**Comment by userquin** [comment]
In the meantime try using external pwa assets config file: https://github.com/vite-pwa/vite-plugin-pwa/tree/main/examples/assets-generator

**Comment by userquin** [comment]
It seems we've a problem when resolving the pwa assets configuration, the disabled flag is false and so the module not being loaded.

**Comment by userquin** [comment]
@LoranRendel released new `v0.19.3`, should be fixed with this version
